src/Wallabag/ApiBundle/Controller/WallabagRestController.php

```diff
@@ -14,17 +15,61 @@
 use Symfony\Component\Security\Core\Exception\AccessDeniedException;
 use Wallabag\UserBundle\Entity\User;
 
+class ApplicationInfo
```

Comment:
could you please create a new file for this class?

Reply:
Fixed